clock_exercise/AlarmItem.qml

```diff
@@ -0,0 +1,58 @@
+import QtQuick 2.0
+
+Item {
+    id: root
+
+    width: 425
+    height: 88
+
+    CheckBox {
+        x: 11
+        anchors.verticalCenter: parent.verticalCenter
+        on: selected
+        onChanged: {
+            selected = !selected
+        }
+    }
+
+    Text {
+        id: dateText
+
+        x: 84
+        y: 12
+        text: day.toString().padStart(2, '0') + "/" + month.toString().padStart(
+                  2, '0') + "/" + year.toString().padStart(4, '0')
+        color: "#9FAAB0"
+        font.family: "Buenos Aires"
+        font.styleName: "Light"
+        font.pixelSize: 18
+        visible: !everyday
+    }
+
+    Text {
+        x: dateText.x
+        y: everyday ? 18 : 33
+        text: hours.toString().padStart(2, '0') + ":" + minutes.toString(
+                  ).padStart(2, '0')
+        color: "#9FAAB0"
+        font.family: "Buenos Aires"
+        font.styleName: "Regular"
+        font.pixelSize: 40
+    }
+
+    ActivationAlarmSwitch {
```

Comment:
Anche in questo caso avrei scelto un nome più semplice come ad esempio `AlarmSwitch`.
La parola `Activation` in questo caso non aggiunge molto significato alla componente, considerando che il ruolo di uno switch è implicitamente quello di attivare/disattivare qualcos'altro.